packages/ui/__stories__/TextArea.stories.tsx

```diff
@@ -0,0 +1,121 @@
+const text =
+  'Yoda, a Force-sensitive male being belonging to a mysterious species, was a legendary Jedi Master who witnessed the rise and fall of the Galactic Republic, followed by the rise of the Galactic Empire. Small in stature but revered for his wisdom and power, Yoda trained generations of Jedi, ultimately serving as the Grand Master of the Jedi Order. Having lived through nine centuries of galactic history, he played integral roles in the Clone Wars, the rebirth of the Jedi through Luke Skywalker, and unlocking the path to immortality.'
+
+export default {
+  title: 'Components/TextArea',
+  component: TextArea,
+}
+
+export const Default = () => (
+  <TextArea
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+  />
+)
+
+export const Empty = () => (
+  <TextArea
+    name="name"
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+  />
+)
+
+export const Error = () => (
+  <TextArea
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+    error="Dark side"
+  />
+)
+
+export const Hovered = () => (
+  <TextArea
+    className={styles.hover}
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+  />
+)
+
+export const Focused = () => (
+  <TextArea
+    className={styles.focus}
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+  />
+)
+
+export const FocusedWithError = () => (
+  <TextArea
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+    // eslint-disable-next-line jsx-a11y/no-autofocus
+    className={styles.focus}
+    error="Dark side"
+  />
+)
+
+export const Disabled = () => (
+  <TextArea
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+    disabled
+  />
+)
+
+export const NotResizable = () => (
+  <TextArea
+    name="name"
+    value={text}
+    placeholder="Describe character"
+    label="Character description"
+    onBlur={() => logger.log('blur')}
+    onChange={(e) => logger.log('change', e.target.value)}
+    resizable={false}
+  />
+)
+
+export const WithHelperText = () => (
```

Comment:
When focusing a textarea with helper text I see two focus borders. + if I hover over help, I don't expect hover to be activated.

![Screenshot 2020-11-12 at 10 08 59](https://user-images.githubusercontent.com/1083817/98919572-45094e00-24cf-11eb-8749-e75e249631b4.png)
![Screenshot 2020-11-12 at 10 08 56](https://user-images.githubusercontent.com/1083817/98919575-463a7b00-24cf-11eb-9bad-7b6eec284f6a.png)

Reply:
Quite a catch!

Reply:
The two outlines appear every time if you try to make text area smaller in width.
![image](https://user-images.githubusercontent.com/17862968/98921376-83a00800-24d1-11eb-846f-ef4ac5fedf2f.png)

Reply:
The scrollbar is overlapping round corner borders on chrome, linux:
![image](https://user-images.githubusercontent.com/17862968/98921880-150f7a00-24d2-11eb-814d-0c479a0f5f76.png)
![image](https://user-images.githubusercontent.com/17862968/98921967-2eb0c180-24d2-11eb-82b3-ad6dab6768d9.png)

Reply:
Fixed in https://github.com/hazelcast/frontend-shared/pull/62/commits/356c037e0d2083ee516ebbf607e8bf56de0eb669